ndc-reference/universities.json

```diff
@@ -0,0 +1,2 @@
+{"id": 1, "name": "Queen Mary University of London", "location": { "city": "London", "country": "UK"}, "professors": [{"id": 1, "first_name": "Peter", "last_name": "Landin"}]}
+{"id": 2, "name": "Chalmers University of Technology", "location": { "city": "Gothenburg", "country": "Sweden"}, "professors": [{"id": 2, "first_name": "John", "last_name": "Hughes"}, {"id": 3, "first_name": "Koen", "last_name": "Claessen"}]}
```

Comment:
ID 3 doesn't appear in `authors` if this is meant to act like a foreign key?

Reply:
I see why you picked this, but it'd be nice if we could come up with some data format which exercised all code paths, without repeating the data.